cleanup issue 43 (#401)
ietf-wg-gnap/gnap-core-protocol: draft-ietf-gnap-core-protocol.md

The AS SHOULD NOT re-use subject identifiers for multiple different ROs.

> Note: the "formats" and "assertions" request fields are independent of each other, and a returned assertion MAY use a different subject identifier than specified in the request, as long as it refers to the same person.

Reviewer: ```suggestion
The "formats" and "assertions" request fields are independent of each other, and a returned assertion MAY use a different subject identifier than other assertions and subject identifiers in the response. All subject identifiers and assertions returned MUST refer to the same person.
```
